src/test/scala/org/bitcoins/core/protocol/script/CLTVScriptPubKeySpec.scala

```diff
@@ -0,0 +1,14 @@
+package org.bitcoins.core.protocol.script
+
+import org.bitcoins.core.gen.ScriptGenerators
+import org.scalacheck.{Properties, Prop}
+
+/**
+  * Created by tom on 8/23/16.
+  */
+class CLTVScriptPubKeySpec extends Properties("CLTVScriptPubKeySpec") {
+  property("Serialization symmetry") =
```

Comment:
Try to think of a way to write a property that says if the locktime is NOT in the past, our CLTV script should fail